qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1 Send CV.feature

```diff
@@ -0,0 +1,185 @@
+Feature: 3.1 Send CV.feature
+
+Background:
+Given user opened the page with the form to send CV
+
+Scenario Outline: 3.1_01 Different cases entering data in fields "Name", "Email", attach CV, mark checkbox
+When user enters name <Name>
+And user enters email <Email>
+And user <attach_notattach> CV <CV>
+And user <mark_notmark> checkbox <checkbox>
+And user pushes the "Send" button
+Then result <result> should be <error_noerror>
+Examples:
+|Name   |Email  |CV        |checkbox|result  |
+|valid  |valid  |attach    |mark    |no error|
+|valid  |valid  |attach    |not mark|error   |
+|valid  |valid  |not attach|mark    |error   |
+|valid  |invalid|attach    |mark    |error   |
+|invalid|valid  |attach    |mark    |error   |
+|valid  |invalid|not attach|not mark|error   |
+|invalid|valid  |not attach|not mark|error   |
+|invalid|invalid|attach    |not mark|error   |
+|invalid|invalid|not attach|mark    |error   |
+|invalid|invalid|not attach|not mark|error   |
+|valid  |valid  |not attach|not mark|error   |
+|valid  |invalid|attach    |not mark|error   |
+|valid  |invalid|not attach|mark    |error   |
+|invalid|invalid|attach    |mark    |error   |
+|invalid|valid  |not attach|mark    |error   |
+|invalid|valid  |attach    |not mark|error   |
+
+Scenario: 3.1_02 Invalid lendth data in the field "Name"
+When user enters "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras tempor rhoncus convallis. Mauris tristique ex et sapien egestas,- non malesuada nisl porttitor" in the "Name" input
+And user pushes the "Send" button
+Then options inputs should have "The maximum number of characters is 70" validation error
+And a "red" check mark indicator appears in the "Name" input
+
+Scenario: 3.1_03 Entering the maximum number of characters (70) in the field "Name"
+When user enters "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras tempor r" in the "Name" input
+And user pushes the "Send" button
+Then options inputs should not have validation error
+And a "green" check mark indicator appears in the "Name" input
+
+Scenario: 3.1_04 Entering 69 characters in the field "Name"
+When user enters "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras tempor" in the "Name" input
+And user pushes the "Send" button
+Then options inputs should not have validation error
+And a "green" check mark indicator appears in the "Name" input
+
+Scenario Outline: 3.1_05 Entering one or two words in the field "Name" with different amount of letters inside
+When user enters words <words> consist of letters <letters> in the "Name" input
+And user pushes the "Send" button
+Then result <result> should be <error_noerror> in the "Name" input
```

Comment:
Then user should see result (corresponding result field in example) in the "Name" input

Reply:
Thanks! Done